components/signup/SkillsPage.tsx

```diff
@@ -43,7 +43,7 @@ export default ({ student }): JSX.Element => {
           <span style={{ fontWeight: 'bold' }}>Skills</span>
           <span style={{ color: COLORS.GRAY_BB }}> that you would use in your projects</span>
         </Typography>
-        <ArrayForm label="Skills and Tools" items={skills} setItems={setSkills} />
+        <ArrayForm label="Skills and Tools" items={skills} setItems={setSkills} options={skills} />
```

Comment:
- need to add options prop in line 30, then pass that into ArrayForm's options
- modify pages/signup/setup to fetch from backend (like you did the filter pages)